docs/reference/commandline/volume_create.md

```diff
@@ -27,7 +27,7 @@ Options:
   -o, --opt value       Set driver specific options (default map[])
 ```
 
-Creates a new volume that containers can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:
+Create a new volume that containers can consume and store data in. If a name is not specified, Docker generates a random name. You create a volume and then configure the container to use it, for example:
```

Comment:
Wait: @yuexiao-wang did you just push again? Looks like this was changed back again